Modules/Field/circular_field.py

```diff
@@ -0,0 +1,24 @@
+import math
+
+from . import config
+
+def circular_field(angle:float):
```

Comment:
Definitely this is not UML oriented. You can ignore this comment for now, but we will have to restructure the algorithms & math system structure.